CHARTER.adoc

```diff
@@ -0,0 +1,203 @@
+
+=== Specific Existing Work Used as Starting Point
+* GeoZarr draft specification: https://github.com/zarr-developers/geozarr-spec/ 
+
+=== Is This a Persistent SWG
+
+[x] YES
+
+[ ] NO
+
+=== When can the SWG be Inactivated
+
+The SWG can be inactivated once the SWG identifies no new tasks for the SWG and there are no open Change Requests.
+
+== Description of deliverables
+The GeoZarr SWG will deliver a candidate Standard and associated developer resources.
+
+The SWG expects to have a candidate Standard ready for OGC Architecture Board (OAB) review and public comment within nine months of creation of the SWG. Because example implementations will be developed at the same time the candidate Standard is formalized, reference implementations that fully use GeoZarr should be documented at the same time the candidate Standard goes to vote.
+
+=== Initial Deliverables
+
+The following deliverables will be the initial results of work of the SWG.
+
+* OGC GeoZarr Standard
+
+* GeoZarr developer resources
+
+The targeted start date for this SWG is 
+
+=== Additional SWG Tasks
+
+No specific additional tasks are currently planned for the SWG.
+
+== IPR Policy for this SWG
+
+[x] RAND-Royalty Free
+
+[ ] RAND for fee
+
+== Anticipated Audience / Participants
+
+This SWG will develop a Standard for general use in the geospatial community and suitable for data exchange beyond this community. Geospatial data providers and software implementers will be interested in assisting with the development of this Standard as well as the output of the SWG.
+
+== Domain Working Group Endorsement
+
+The SWG convenors will discuss the charter with potentially interested Domain Working Groups (DWGs) at the first opportunity.
+
+== Other informative information about the work of this SWG
+
+=== Collaboration
+
+All work in the Standards Working Group will be public and the SWG solicits contributions and feedback from OGC members and non-OGC members to the extent that is supported by the OGC Technical Committee Policies and Procedures.
+
+The OGC GeoZarr SWG will collaborate on Standard development using a public GitHub repository and a Gitter channel. Development of the Standard will include the use of Issues and other project tools in GitHub.
+
+=== Similar or Applicable Standards Work (OGC and Elsewhere)
+
+* The OGC endorsed a community standard of Zarr V2.0 (https://zarr.readthedocs.io/en/stable/spec/v2.html) in June 2022.
+
+* This SWG is closely related to the newly announced [Geodatacube SWG](https://www.ogc.org/press-release/ogc-forms-new-geodatacube-standards-working-group/). Essentially, Geodatacube will specify a server API while GeoZarr will define a standard Cloud-native format for a serverless datacube. Therefore, close coordination between these SWGs seems needed.
```

Comment:
Suggest "GeoZarr will define" -> "GeoZarr can define" since GeoZarr can be used for data structures other than datacubes (as described earlier in the Mixing Data bullet).